src/components/pages/python.tsx

```diff
@@ -45,10 +50,32 @@ function Python(): JSX.Element {
       { ['print']: Color.Orange },
       { ['(color']: Color.White },
       { [' != ']: Color.Orange },
-      { ['"red")']: Color.White},
+      { ['"red"']: Color.Blue},
+      { [')']: Color.White},
     ],
   ];
 
+  const [top, setTop] = useState(false);
+  const [middle, setMiddle] = useState(false);
+  const [bottom,setBottom] = useState(false);
```

Comment:
can we make this a single state? then we can squish 62-77. it'll streamline the code a bit + easier to make bug fixes 

let me know if you want more details/ideas!